lib/main.dart

```diff
@@ -37,8 +36,8 @@ class HomePage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    context.read<AuthBloc>().add(const AuthEventInitialize());
-    return BlocConsumer<AuthBloc, AuthState>(
+    context.read<AuthCubit>().initialize();
```

Comment:
let's not call init() in build method, build methods can be frequently rebuilt

call it when cubit it created